wradlib/io/iris.py

```diff
@@ -132,8 +132,28 @@ def decode_array(
 
 
 def decode_rainrate2(data):
```

Comment:
We can completely remove this function. All decoding can be done in `decode_array`.